specs/APIReview_AllowExternalDrop.md

```diff
@@ -0,0 +1,100 @@
+# Background
+Currently the functionality of dragging and dropping external objects(e.g. files, hyperlinks) into webview2 is default enabled in webview2 and there is no way to disable it. Some developers may want to disbale this functionality in their applications based upon their scenarios. According to such feature requirements, we add the new API to provide developers with the capability to configure the external drag&drop functionality.
```

Comment:
The text here still requires editing/wordsmithing for spelling/smoothness.

Reply:
```suggestion
Currently the functionality of dragging and dropping external objects (e.g. files, hyperlinks) into webview2 is by default enabled in webview2 and there is no way to disable it. Some developers may want to disable this functionality in their applications. To allow for this, we add the new API to provide developers with the capability to disable the external drag & drop functionality.
```

Reply:
Thanks @david-risney, changed to proposed text.